src/asset.js

```diff
@@ -1,66 +1,121 @@
-const asset = {
-  main: {
-    name: '10 Monkey Species Small',
-    dateCreated: '2012-02-01T10:55:11Z',
-    author: 'Mario',
-    type: 'dataset',
-    license: 'CC0: Public Domain',
-    price: '0',
-    files: [
-      {
-        index: 0,
-        contentType: 'application/zip',
-        checksum: '2bf9d229d110d1976cdf85e9f3256c7f',
-        checksumType: 'MD5',
-        contentLength: '12057507',
-        compression: 'zip',
-        encoding: 'UTF-8',
-        url:
-          'https://s3.amazonaws.com/datacommons-seeding-us-east/10_Monkey_Species_Small/assets/training.zip'
-      },
-      {
-        index: 1,
-        contentType: 'text/txt',
-        checksum: '354d19c0733c47ef3a6cce5b633116b0',
-        checksumType: 'MD5',
-        contentLength: '928',
-        url:
-          'https://s3.amazonaws.com/datacommons-seeding-us-east/10_Monkey_Species_Small/assets/monkey_labels.txt'
+
+
+   export function getAsset(){
```

Comment:
why is this a function? Keeping it as `const asset =` should have the same effect, no? Again, it increases the difficulty of the tutorial https://docs.oceanprotocol.com/tutorials/react-publish-data-set/#define-asset

Reply:
One way to solve the difficulty issue would be to isolate the compute stuff into another file like `asset-compute.js`, and keep `asset.js` in its simple form.